api/apps/search/routes.js

```diff
@@ -5,7 +5,7 @@ const { NODE_ENV } = process.env
 
 // GET /api/search
 export const index = (req, res, next) => {
-  const env = NODE_ENV === 'production' ? 'production' : 'staging'
+  const env = NODE_ENV === 'production' ? 'production' : 'production'
```

Comment:
Should this be changed back to staging?

Reply:
yes, sorry, fixed that, wanted to try with production locally.